### DISCUSSION TITLE
How Can We Enable Brotli Compression?

### DISCUSSION THREAD

**Discussion by darrylmorley** [caprover/caprover#2091] [Q&A] [answered]
As in the title, is it possible to enable Brotli? Can the main Nginx image be swapped out for one with Brotli already included?

Thanks

**Comment by githubsaturn** [comment, marked as answer]
Yes, but it's an advance process:
https://caprover.com/docs/troubleshooting.html#customize-config-settings

**Reply by darrylmorley** [reply]
Great, thanks for the pointer 👍
